src/include/basil.h

```diff
@@ -69,230 +68,193 @@
-#define BASIL_ATR_GROUP_ID	"group_id"	/* BASIL_ELM_APPLICATION */
-#define BASIL_ATR_TIME_STAMP	"time_stamp"	/* BASIL_ELM_APPLICATION */
-#define BASIL_ATR_MEMORY	"memory"	/* BASIL_ELM_COMMAND */
-#define BASIL_ATR_MEMORY_MB	"memory_mb"	/* BASIL_ELM_ACCELERATOR */
-#define BASIL_ATR_NODE_COUNT	"node_count"	/* BASIL_ELM_COMMAND */
-#define BASIL_ATR_COMMAND	"cmd"		/* BASIL_ELM_COMMAND */
-#define BASIL_ATR_SEGMENT_ID	"segment_id"	/* BASIL_ELM_RSVD_SGMT */
-#define BASIL_ATR_FAMILY	"family"	/* BASIL_ELM_ACCELERATOR */
-#define BASIL_ATR_ACTION	"action"	/* BASIL_ELM_APPLICATION */
-/* BASIL_ELM_RESERVATION */
-#define BASIL_ATR_PGOVERNOR	"p-governor"	/* BASIL_ELM_RESERVEPARAM */
-#define BASIL_ATR_PSTATE	"p-state"	/* BASIL_ELM_RESERVEPARAM */
+#define BASIL_ATR_PROTOCOL	"protocol"		/* BASIL_ELM_REQUEST */
+							/* BASIL_ELM_RESPONSE */
+#define BASIL_ATR_METHOD	"method"		/* BASIL_ELM_REQUEST */
+							/* BASIL_ELM_RESPONSEDATA */
+#define BASIL_ATR_STATUS	"status"		/* BASIL_ELM_RESPONSEDATA */
+							/* BASIL_ELM_APPLICATION */
+							/* BASIL_ELM_RESERVATION */
+#define BASIL_ATR_ERROR_CLASS	"error_class"		/* BASIL_ELM_RESPONSEDATA */
+#define BASIL_ATR_ERROR_SOURCE	"error_source"		/* BASIL_ELM_RESPONSEDATA */
+#define BASIL_ATR_SEVERITY	"severity"		/* BASIL_ELM_MSG */
+#define BASIL_ATR_TYPE		"type"			/* BASIL_ELM_REQUEST:query */
+							/* BASIL_ELM_MEMORY */
+							/* BASIL_ELM_LABEL */
+							/* BASIL_ELM_ACCELERATOR */
+#define BASIL_ATR_USER_NAME	"user_name"		/* BASIL_ELM_RESERVEARRAY */
+							/* BASIL_ELM_RESERVATION */
+#define BASIL_ATR_ACCOUNT_NAME	"account_name"		/* BASIL_ELM_RESERVEARRAY */
+							/* BASIL_ELM_RESERVATION */
+#define BASIL_ATR_BATCH_ID	"batch_id"		/* BASIL_ELM_RESERVEARRAY */
+							/* BASIL_ELM_RESERVATION */
+#define BASIL_ATR_PAGG_ID	"pagg_id"		/* BASIL_ELM_RESERVATION */
+							/* BASIL_ELM_REQUEST:confirm */
+							/* BASIL_ELM_REQUEST:cancel */
+#define BASIL_ATR_ADMIN_COOKIE	"admin_cookie"		/* synonymous with pagg_id */
+#define BASIL_ATR_ALLOC_COOKIE	"alloc_cookie"		/* deprecated as of 1.1 */
+#define BASIL_ATR_CHANGECOUNT	"changecount"		/* BASIL_ELM_NODEARRAY */
+#define BASIL_ATR_SCHEDCOUNT	"schedchangecount"	/* BASIL_ELM_SUMMARY */
+							/* BASIL_ELM_NODEARRAY */
+#define BASIL_ATR_CLAIMS	"claims"		/* BASIL_ELM_RELEASED */
+#define BASIL_ATR_RSVN_ID	"reservation_id"	/* BASIL_ELM_RESERVATION */
+							/* BASIL_ELM_REQUEST:confirm */
+							/* BASIL_ELM_REQUEST:release */
+#define BASIL_ATR_JOB_NAME	"job_name"		/* BASIL_ELM_REQUEST:confirm */
+#define BASIL_ATR_NODE_ID	"node_id"		/* BASIL_ELM_NODE */
+#define BASIL_ATR_ROUTER_ID	"router_id"		/* BASIL_ELM_NODE */
+#define BASIL_ATR_ARCH		"architecture"		/* BASIL_ELM_RESERVE */
+							/* BASIL_ELM_NODEARRAY */
+							/* BASIL_ELM_NODE */
+							/* BASIL_ELM_PROCESSOR */
+							/* BASIL_ELM_COMMAND */
+#define BASIL_ATR_ROLE		"role"			/* BASIL_ELM_NODE */
+#define BASIL_ATR_WIDTH		"width"			/* BASIL_ELM_RESERVEPARAM */
```

Comment:
@lisa-altair and @dilip-krishnan may be this can you help you guys: https://github.com/dilip-krishnan/pbspro/blob/PP-809/src/include/basil.h?ts=4 and https://github.com/dilip-krishnan/pbspro/blob/PP-809/src/include/basil.h?ts=8

(And FYI https://github.com/tiimgreen/github-cheat-sheet#readme)